core/governance/engine_batch.go

```diff
@@ -331,13 +341,14 @@ func (e *Engine) addBatchVote(ctx context.Context, batchProposal *batchProposal,
 	}
 
 	vote := types.Vote{
-		PartyID:                     party,
-		ProposalID:                  cmd.ProposalID,
-		Value:                       cmd.Value,
-		Timestamp:                   e.timeService.GetTimeNow().UnixNano(),
-		TotalGovernanceTokenBalance: getTokensBalance(e.accs, party),
-		TotalGovernanceTokenWeight:  num.DecimalZero(),
-		TotalEquityLikeShareWeight:  num.DecimalZero(),
+		PartyID:                        party,
+		ProposalID:                     cmd.ProposalID,
+		Value:                          cmd.Value,
+		Timestamp:                      e.timeService.GetTimeNow().UnixNano(),
+		TotalGovernanceTokenBalance:    getTokensBalance(e.accs, party),
+		TotalGovernanceTokenWeight:     num.DecimalZero(),
+		TotalEquityLikeShareWeight:     totalELSWeight,
```

Comment:
I don't think `totalELSWeight` being the sum of the per-market one makes much sense. Can we just leave it as zero? and in `governance.proto` say how that `TotalEquityLikeShareWeight` is only populated if it was a vote on a single-proposal and `PerMarketEquityLikeShareWeight` is only populated if its a batch proposal?

Reply:
Seems to be addressed.